metaschema/generate.go

```diff
@@ -31,7 +39,8 @@ func main() {
 
 	metaschemaPaths := []string{
 		"oscal-catalog-metaschema.xml",
-		"oscal-profile-metaschema.xml",
+		"oscal-ssp-metaschema.xml",
+		//	"oscal-profile-metaschema.xml",
```

Comment:
Any reason for commenting this one out?

Reply:
`make generate` is failing because profile's metaschema imports a filepath that does not exist in the NIST OSCAL Repository.